src/helpers/SocialReferrer.js

```diff
@@ -4,20 +4,28 @@ var SocialReferrer = {
    *
    * @returns {String};
    */
-  getReferrer: function() {
+  getReferrer: function () {
     return window.document.referrer || '';
   },
  /**
-  * Determines if the referrer is social media
+  * Checks if the referrer is social media.
+  * If true, we want to return the type of social media
+  * else return false
   *
-  * @returns {Boolean};
+  * @returns {String};
   */
-  isSocialReferrer: function() {
-    var socialReferrer = this.getReferrer().match(/\b(?:facebook|instagram|pinterest|reddit|twitter|tumblr|t\.co)\b/i);
+  getSocialReferrer: function () {
+    // brandMap is used to match when it is a twitter short url
+    var socialReferrer = this.getReferrer().match(/\b(?:facebook|instagram|pinterest|reddit|twitter|tumblr|t\.co)\b/i),
+      brandMap = {
+        't.co': 'twitter'
+      };
     if (socialReferrer) {
-      return true;
+      // if the match is positive, return the first result and check if it is a twitter short url
+      socialReferrer = socialReferrer[0].toLowerCase();
```

Comment:
String.match() is kind of unintuitive in that it returns in an array on success.  Would it be clearer if you checked for a match as the condition of the `if` and then returned `this.getReferrer().toLowerCase()` if it was true?. 

In a semi-related thought, would it makes sense to define a function that has all the various potential social media values set as variables that get checked rather than lumping them into a single regex expression?